lib/client-assets.php

```diff
@@ -97,7 +97,7 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'utils/build/index.js' ),
 		true
 	);
-	wp_add_inline_script( 'wp-utils', 'var originalUtils = wp.utils;', 'before' );
+	wp_add_inline_script( 'wp-utils', 'window.wp = window.wp || {}; var originalUtils = wp && wp.utils ? wp.utils : {};', 'before' );
```

Comment:
Do we need the first part of this line? `window.wp = window.wp || {}; `

Reply:
Well since the error is `ReferenceError: Can't find variable: wp` I believe so. If the `wp-utils` is the first script to be loaded, there will be no `wp` global defined at all before the script.

Reply:
Oh, right. Though then it begs the question do we need the second part, `wp &&`, since we know it'll be defined since we just defined it if it wasn't.

Maybe we just test against the property on `window`:

```js
var originalUtils = window.wp && window.wp.utils ? window.wp.utils : {};
```